src/components/Outcomes/OutcomeGenerator/OutcomeGenerator.tsx

```diff
@@ -32,11 +42,31 @@ export const OutcomeGenerator: FC<OutcomeGeneratorProps> = () => {
         <RichTextOutput content={gameInfoContext.outcome.productsIntro} />
       </Text>
       {requiredProducts && requiredProducts.length > 0 && (
-        <List size="lg" pb="20px" id='required-products'>
+        <Accordion size="lg" pb="20px" id="required-products" allowMultiple>
           {requiredProducts.map((product: TargetProduct) => (
-            <ListItem>{product}</ListItem>
+            <AccordionItem>
```

Comment:
The new AccordionItem likely breaks how the e2e tests check that the proper number of products is displayed on a page. This PR was made before we got the fix in that properly ran e2e tests so only the component tests ran against this PR. 

I can make the required change to the e2e tests. Unless you want to.

Reply:
Happy to let you make whatever changes are required for the e2e tests so that we can merge safely.